assets/css/bsky_comments.css

```
@@ -1,122 +1,117 @@
+    color: var(--mut);
+    font-size: 0.9rem;
 }
 
-/* Comment content */
 .bsky-content {
-	margin: 0.5rem 0;
-	line-height: 1.5;
-	word-wrap: break-word;
+    margin: 0.5rem 0;
+    line-height: 1.5;
+    word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)